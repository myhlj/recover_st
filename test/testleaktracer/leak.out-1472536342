# LeakTracer report diff_utc_mono=1472453735.879778
leak, time=82606.755454, stack=0x7fb08444df00 0x7fb084be14ea 0x7fb084be15fb 0x7fb084bd1cfa, size=72704, data=..................................................
leak, time=82606.755526, stack=0x7fb0847467dc 0x400a60 0x7fb08401b830 0x400909, size=100, data=hahahhahah........................................
leak, time=82606.755538, stack=0x7fb084068185 0x7fb0840764c4 0x7fb084075828 0x7fb0840741bd 0x7fb0840696fb, size=1024, data=ljhas 26 years old................................
