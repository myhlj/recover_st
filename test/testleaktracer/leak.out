# LeakTracer report diff_utc_mono=1472615155.888050
leak, time=13972.479638, stack=0x7fbca6bf4f00 0x7fbca73884ea 0x7fbca73885fb 0x7fbca7378cfa, size=72704, data=..................................................
leak, time=13972.479693, stack=0x7fbca6eed7dc 0x400a60 0x7fbca67c2830 0x400909, size=100, data=hahahhahah........................................
leak, time=13972.479705, stack=0x7fbca680f185 0x7fbca681d4c4 0x7fbca681c828 0x7fbca681b1bd 0x7fbca68106fb, size=1024, data=ljhas 26 years old................................
